### PR TITLE
add ppb op to python

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: pytket/binders/circuit/main.cpp [hunk 1]
```diff
@@ -293,6 +293,11 @@ PYBIND11_MODULE(circuit, m) {
           "Reset", OpType::Reset,
           "Resets the qubit to :math:`\\left|0\\right>`")
       .value("CircBox", OpType::CircBox, "Represents an arbitrary subcircuit")
+      .value(
+          "PhasePolyBox", OpType::PhasePolyBox,
+          "A PhasePolyBox is capable of representing arbitrary Circuits made "
+          "up of CNOT and RZ, as a PhasePolynomial plus a boolean matrix "
+          "representing an additional linear transformation.")
```
Comment (cqc-alec) [inline pytket/binders/circuit/main.cpp:300]:
```suggestion
      .value(
          "PhasePolyBox", OpType::PhasePolyBox,
          "An operation representing arbitrary circuits made "
          "up of CX and Rz gates, represented as a phase polynomial together with a boolean matrix "
          "representing an additional linear transformation.")
```

**Review by cqc-alec — CHANGES_REQUESTED** [review]
Please add line to changelog too.

**Comment by cqc-melf** [comment]
I have added it to the changelog and fixed the format error

File: pytket/docs/changelog.rst [hunk 2]
```diff
@@ -12,6 +12,8 @@ Minor new features:
 * New pass ``SynthesiseTK`` and transform ``OptimiseStandard`` to synthesize
   TK2 gates.
 * Add ``Optype.WASM``, adding a classical wasm function call to the circuit
+* Add optype for existing PhasePolyBox ``Optype.PhasePolyBox``
```
Comment (cqc-alec) [inline pytket/docs/changelog.rst:15]:
```suggestion
* Add optype for existing PhasePolyBox ``OpType.PhasePolyBox``
```

**Pull request merged**